src/styles.scss

```diff
@@ -5,5 +5,12 @@ h1 {
 }
 
 body {
-    background-image: url('../Users/eliseparker/Projects/JS-Class/JS-Project/src/images/logo.jpg');
+    background-color: #ffffff;
 }
+
+.logo{
+    text-indent: -9999999990px;
+    background-image: url('logo.jpg');
```

Comment:
Reverted to using the background-image tag in CSS versus the <img src='path'/> tag in HTML because I had issues with a broken image. Confirmed file paths and filetype were correct.